pkg/schemaregistry/next/composable/cloudwatch/dataquery.cue

```diff
@@ -0,0 +1,149 @@
+// Code generated - EDITING IS FUTILE. DO NOT EDIT.
+//
+// Generated by:
+//     public/app/plugins/gen.go
+// Using jennies:
+//     PluginSchemaRegistryJenny
+//
+// Run 'make gen-cue' from repository root to regenerate.
+
+import (
+	"github.com/grafana/kindsys"
+	"github.com/grafana/grafana/packages/grafana-schema/src/common"
+)
+
+kindsys.Composable & kindsys.Composable & {
+	maturity:        "merged"
+	name:            "CloudWatch"+"DataQuery" & "CloudWatch"+"DataQuery"
```

Comment:
This is clearly a no-op, but since we're thinking about byte-level equality in the checks, it may be a concern. (The check as-written in this PR is only on the contents of schema, but we'll likely need to add more rules related to kind props as we go along)